color/src/parse.rs

```diff
@@ -87,6 +87,22 @@ enum Value<'a> {
     Dimension(f64, &'a str),
 }
 
+/// Whether or not we are parsing modern or legacy mode syntax.
+#[derive(Clone, Copy, Debug, PartialEq)]
+enum Mode {
+    Legacy,
+    Modern,
+}
+
+impl Mode {
```

Comment:
A fun trick you could do here is have `Mode` to be `#[repr(u8)]` and have the values be `b','` and `b'/'`, then this method is just a cast. I'm not sure if that helps with anything :)

Reply:
I bikeshedded this so much ... I had that in an earlier version when this enum was just `OpacitySeparator`, but I didn't want that here where it might (should) control more things than just that...